packages/datagateway-dataview/public/res/default.json

```diff
@@ -122,7 +122,8 @@
   "loading": {
     "verifying": "Verifying URL",
     "oops": "Oops!",
-    "message": "We&apos;re sorry, it seems as though the URL you requested is attempting to fetch incorrect data. Please double check your URL, navigate back via the breadcrumbs or <2>go back to the top level</2>."
+    "message": "We&apos;re sorry, it seems as though the URL you requested is attempting to fetch incorrect data. Please double check your URL, navigate back via the breadcrumbs or <2>go back to the top level</2>.",
+    "filter_message": "No results found with current filters applied, please modify filter settings and try again."
```

Comment:
The message gets displayed when filtering for absent data, however if you toggle to the table view and then toggle back to the card view, you will not see the message. 

Not related to this PR but when filtering for data that is present in one view and then toggling to the other view, the filter is not preserved. For example, if you filter by _dog_ in the card view then toggle to the table view, the table does not contain the filtered results but all of the results instead.

Reply:
Got the first part working. The point about the seperate filters for CV/table is on the list of things to address, but I didn't want to do too much in one PR so was trying to stick to purely UI issues like alignment. It'll be covered at in a later PR I expect along with some other querying issues.

Reply:
>The point about the seperate filters for CV/table is on the list of things to address, but I didn't want to do too much in one PR so was trying to stick to purely UI issues like alignment. It'll be covered at in a later PR I expect along with some other querying issues.

That's perfectly reasonable, I just wanted to flag it so that we are aware of it.